.github/workflows/test-python.yml

```diff
@@ -43,8 +43,8 @@ jobs:
       - name: Checkout Django
         uses: actions/checkout@v4
         with:
-          repository: 'mongodb-forks/django'
-          ref: 'mongodb-5.0.x'
+          repository: 'wavev/django'
```

Comment:
The test failure is because of some new test on the Django fork. I merged in the changes from your branch, so you can use mongodb-5.0.x again.

Incidentally, can you structure this as two commits:
1) "add generic_relations tests to CI"
2) the fix (which removes the expected test failures added in commit 1 so it's more clear what's fixed)

Reply:
Yeah, solved.